src/Stratis.SmartContracts.Core/State/AccountAbstractionLayer/TransferInfo.cs

```diff
@@ -9,6 +9,13 @@ namespace Stratis.SmartContracts.Core.State.AccountAbstractionLayer
     /// </summary>
     public class TransferInfo
     {
+        public TransferInfo(uint160 from, uint160 to, ulong value)
+        {
+            this.From = from;
+            this.To = to;
+            this.Value = value;
+        }
+
         public uint160 From { get; set; }
```

Comment:
Hmm, immutable? Should the set; be removed?

Reply:
:joy: probably, that was the whole point of this PR